modules/sdk-coin-ada/src/ada.ts

```diff
@@ -105,7 +106,11 @@ export class Ada extends BaseCoin {
   }
 
   isWalletAddress(params: VerifyAddressOptions): boolean {
-    throw new Error('Method not implemented.');
+    const { address } = params;
+    if (!this.isValidAddress(address)) {
+      throw new InvalidAddressError(`Invalid Cardano Address: ${address}`);
+    }
+    return true;
```

Comment:
Shouldn't we also be checking if this address belongs to a wallet?

Reply:
Good question! @ewangbitgo can you confirm this? When I looked at other coins, this method was either not implemented, or just checked if it was a valid address.